internal/fpath/path.go

```diff
@@ -12,7 +12,11 @@ import (
 	"strings"
 )
 
-const storjScheme = "sj"
+// Create a set
+var storjScheme = map[string]struct{}{
```

Comment:
just curious why `map[string]struct` ?

Reply:
not blocking btw

Reply:
It is saving resources and improves performance. Kaloyan left a good post: https://emersion.fr/blog/2017/sets-in-go/

Reply:
Instead of having a byte for an bool

Reply:
nice! I've always just used `map[string]bool` but will now be switching to `map[string]struct` great write up @kaloyan-raev